civitai-downloader-v2/tests/api/test_websocket.py

```
@@ -0,0 +1,312 @@
+
+class TestWebSocketErrorHandling:
+    """Test WebSocket error handling and reconnection."""
+    
+    def test_websocket_connection_error(self, test_client: TestClient):
+        """WebSocket接続エラーが適切に処理されることを確認"""
+        # Test connection to invalid endpoint
+        try:
+            with test_client.websocket_connect("/invalid-ws-endpoint"):
+                pass
+            # Should not reach here
+            assert False, "Expected connection error"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)